openupgrade_scripts/scripts/auth_totp/15.0.1.0/post-migration.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # This method will copy records has scope = '2fa_trusted_device from res_users_apikeys to auth_totp_device
+    # Then change value of scope to 'browser'
+    openupgrade.logged_query(
+        env.cr,
+        """INSERT INTO auth_totp_device (id, name, user_id, scope, index, key)
+            SELECT id, name, user_id, '%s', index, key 
```

Comment:
@daonamutc1 thừa dấu cách ở cuối, tiện anh rebasse luôn nhé
```suggestion
            SELECT id, name, user_id, '%s', index, key
```